modules/hc_resource/config/install/field.field.node.hc_resource.field_hc_hide_header_image.yml

```diff
@@ -3,20 +3,17 @@ status: true
 dependencies:
   config:
     - field.storage.node.field_hc_hide_header_image
-    - node.type.hc_article
-_core:
-  default_config_hash: d7tLfdS-UsSpR_Ao6GOOZp5hU05OlNqjbooVed0CFII
-id: node.hc_article.field_hc_hide_header_image
+    - node.type.hc_resource
+id: node.hc_resource.field_hc_hide_header_image
 field_name: field_hc_hide_header_image
 entity_type: node
-bundle: hc_article
+bundle: hc_resource
 label: 'Hide Header Image'
-description: 'Choose this option if you wish to hide the header image for this page.'
+description: ''
```

Comment:
Why are we removing the description here?

Reply:
Restored in later commit